fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -97,7 +101,11 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 		return Void();
 	}
 	ACTOR Future<Void> _start(Database cx, SpecialKeySpaceCorrectnessWorkload* self) {
-		if (self->clientId == 0) wait(timeout(self->getRangeCallActor(cx, self), self->testDuration, Void()));
+		if (self->clientId == 0) {
```

Comment:
I'm thinking: Do we have to run the `testConflictRanges()` test with only 1 client? From my understanding, it seems possible to run multiple clients to test this. 

It does not have to be done in this PR though. 

@zjuLcg I feel that this `getRangeCallActor ()`  does not do much for the `\xff\xff/transaction/conflicting_keys/` special key space. I assume you are working on that?

Reply:
If we can test this without the constraint `if (self->clientId == 0)`, it will be great to open an issue and separate PR for that.

Reply:
Yeah. It is definitely better to run with multiple clients. My assumption was that the test is running enough time so one client and others should be similar. But I think you are right, we can definitely allow multiple clients.
And for `\xff\xff/transaction/conflicting_keys/`, it is tested in `ReportConflictingKeys.actor.cpp`. So this is not much related to that.

Reply:
I think the idea for `if (self->clientId == 0)` was to have precise control of when a conflict happens. If we add a unique prefix to keys for each client we could remove it

Reply:
> it is tested in ReportConflictingKeys.actor.cpp

Actually yea I was thinking of a different test. I think we could just run it in all clients. I can follow up on that

Reply:
We can merge this pr first and I can add a pr to solve this and also add unique prefixes in `conflictingKeys` workload to improve coverage.

Reply:
Sounds good.